lib/components/EmailInput.js

```diff
@@ -109,14 +112,18 @@ EmailInput.propTypes = {
    * To specify the text to be displayed inside the input field.
    */
   placeholder: PropTypes.string,
+  /**
+   * The helper text message to be displayed below the input field.
+   */
+  helpText: PropTypes.string,
   /**
    * The values to be displayed inside the input field.
    */
   value: PropTypes.array,
   /**
-   * To specify the error message to be shown in the input field.
+   * To specify the error message to be shown below the input field.
    */
-  error: PropTypes.string,
+  error: PropTypes.oneOfType([String, Array]),
```

Comment:
Let's keep only the string type for the error prop. Please make the changes as discussed over the call. Let me know if there is any difficulty with that.